server/src/main/java/com/cloud/agent/manager/allocator/impl/FirstFitAllocator.java

```diff
@@ -203,8 +203,7 @@ public List<Host> allocateTo(VirtualMachineProfile vmProfile, DeploymentPlan pla
 
 
         if (clusterHosts.isEmpty()) {
-            logger.error("No suitable host found for vm [{}] with tags [{}].", vmProfile, hostTagOnOffering);
-            throw new CloudRuntimeException(String.format("No suitable host found for vm [%s].", vmProfile));
+            logger.warn("No suitable host found for VM [{}] with tags {} in {}.", vmProfile, hostTagOnOffering, paramAsStringToLog);
```

Comment:
it looks like `return null;` appears in #8611 but is missing 
@Pearl1594

Reply:
my bad - done